### DISCUSSION TITLE
Uninstall quietly

### DISCUSSION THREAD

**Discussion by mephistophen** [valinet/ExplorerPatcher#2007] [Ideas]
Hello, do you know how to launch de command of uninstall ep_setup.exe /uninstall but quietly ?

I have to launch multiple uninstall of dozen of workstations because of the issue with Windows Update.

Thank you

**Comment by psydex** [comment]
How do you uninstall this mod at all?

**Reply by Poopooracoocoo** [reply]
https://github.com/valinet/ExplorerPatcher/wiki#uninstalling

**Reply by yougs21** [reply]
In the 3 ways to uninstall, there is a validation window then?

**Reply by mephistophen** [reply]
yes and it's impossible to run the cmd remotly.

**Reply by thangarajua** [reply]
How to uninstall ExplorerPatcher silently?

**Reply by pyrates999** [reply]
No way to silently uninstall it.  Install and Uninstall requires elevated privileges.

**Reply by thangarajua** [reply]
We have to remove this on multiple devices. I found a way to remove it silently but it's prompting a successfully uninstalled message at the end and unable to suppress that.

**Reply by pyrates999** [reply]
Please wait for a developer to respond then.  If the message that is shown when it is successfully uninstalled can be disabled through a command line switch, it will still require elevated privileges to run.

**Comment by nobodyneedsthis** [comment]
I too would like a /silent switch for uninstalling EP.
Running it elevated isn't a problem, the dialog box asking "Are you sure?" is.

**Comment by ioscanner** [comment]
This is a huge issue the 24H2 installer breaks and you can't get to the system except thru command line if you have an RMM.  We need a silent uninstall method.  24H2 doesn't seems to create a restore point so getting this uninstalled is the only way to get around this.

**Comment by TomLiang2021** [comment]
I'm having a problem in uninstall EP and getting win11 to update with 24H2.
My laptop is running on 23H2.
I try a few uninstall tools like Revo, unlocker and EMCO unclock.
None of them works.

I'm keep getting this error.
![image](https://github.com/user-attachments/assets/458595cf-2805-415f-ab13-2a591d216ced)

**Reply by pyrates999** [reply]
just to be sure, did you reboot your pc?

**Comment by tergalphantom** [comment]
Hello, I know this topic is old.
Just to say that I had his problem too with hundreds of PC with black screens to handle, and managed to uninstall somewhat silently ExplorerPatcher with an autoit process.
https://github.com/tergalphantom/EP-Uninstaller

**Reply by pyrates999** [reply]
This repo is an hour old.  And it contains an exe as part of it.  I would hold off on trying this for now.
